lib/services/devices/devices-NGSI-v2.js

```diff
@@ -222,7 +221,7 @@ function updateEntityNgsi2(deviceData, updatedDevice, callback) {
     ) {
         options.json[constants.TIMESTAMP_ATTRIBUTE] = {
             type: constants.TIMESTAMP_TYPE_NGSI2,
-            value: moment()
+            value: new Date().toISOString()
```

Comment:
Really needed or a kind of improvement?

Reply:
Moment.js is deprecated, or rather there are better ways of achieving the same result without the use of an unnecessary extra package. In this case `Date` gives us everything we need from moment.

https://momentjs.com/docs/#/-project-status/

Ideally I'd like to remove both moment and moment-timezone entirely at some point